src/QsCompiler/Tests.Compiler/TestCases/QirEntryPointTests/TestUnitReceivesBool.cpp

```diff
@@ -2,14 +2,171 @@
 // This code was generated by a tool.
 // <auto-generated />
 //------------------------------------------------------------------------------
+//
```

Comment:
Offline, we talked a bit about how to organize tests for this component, just adding a comment such that we don't forget about it.

Reply:
maybe I can do this in a follow up pr?

Reply:
It is usually a good idea to merge implementation and tests in the same PR since it allows for validation before the code is merged. Having said that, I think this PR should contain the tests as well.

Reply:
added tests